### PR TITLE
Cleaner ENV variable support for Docker machines

### PR DESCRIPTION
**Pull request by gregjan**
Removes prior fixes related to Docker and ENV variables for Tamaya
Converts Docker entrypoint to bash, which supported dotted ENV variables.
Adds tamaya.ordinal=500 to the set of ENV variables passed by Docker, giving ENV variables highest preference.
Converts docker config to dotted ENV variables, updating stack config file and added a .env defaults file.
Removes JMS, basic authn, and webac profiles from build. (These were broken anyway against current Trellis)

### CODE REVIEW

File: webapp/pom.xml [hunk 1]
```diff
@@ -100,7 +100,7 @@
       <artifactId>trellis-webdav</artifactId>
       <version>${trellis.version}</version>
     </dependency>
-    <dependency>
+<!--     <dependency>
```
Comment (ajs6f) [inline webapp/pom.xml]: One small question: why comment this out? Do we expect to bring it back at some point?

Comment (gregjan) [reply webapp/pom.xml]: Oh, whoops. No, I left it while running tests, but I'll take it out.

**Comment by gregjan** [comment]
Actually @ajs6f , can you look at the remaining systems properties in the Docker command.sh? Which of those are Tamaya consumed? I may be able to move some more of them into the more visible part of the Docker config.

**Comment by ajs6f** [comment]
Sorry, nothing else there I see. That is all Thorntail stuff. (Which should, in the course of µprofile app development, eventually be normalized into the larger Trellis world of config for Trellis and its container.)

**Comment by gregjan** [comment]
Okay, thought I would double-check. It could be ready for merge then.

**Comment by gregjan** [comment]
I can rebase and test this after #73 merges.

**Comment by gregjan** [comment]
Oh wait, nm.

**Pull request merged**